server/webserver.go

```diff
@@ -749,6 +752,29 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 		w.Write([]byte(fmt.Sprintf("\n\nLatestPlaylist: %v", s.LatestPlaylist())))
 	})
 
+	mux.HandleFunc("/getLogLevel", func(w http.ResponseWriter, r *http.Request) {
+		if vFlag == nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		w.Header().Set("Content-Type", "application/json")
+		w.Write([]byte(vFlag.String()))
+	})
+
+	mux.HandleFunc("/setLogLevel", func(w http.ResponseWriter, r *http.Request) {
+		// URL should be /setLogLevel/[logLevel]
+		if vFlag == nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		err := vFlag.Set(r.FormValue("loglevel"))
```

Comment:
What happens if the form value isn't set?

Reply:
`r.FormValue` will return empty string, `vFlag.Set` will return error and `StatusBadRequest` will be returned to caller.